src/main/java/net/dv8tion/jda/api/entities/emoji/Emoji.java

```diff
@@ -233,9 +233,10 @@ enum Type
          */
         UNICODE,
         /**
-         * Custom Guild Emoji.
+         * Custom Guild Emoji or Custom Application Emoji.
          * <br>This represents emojis which were created by users and added to a guild.
+         * <br>This can also represent emojis which were created by users and added to a jda.
          */
-        CUSTOM,
```

Comment:
Why remove the comma here? I'm sure it was left intentionally.

Reply:
Was an issue with the reformat I did manually. Is fixed in the next commit